recipe/activate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+# For conda-build we add the host requirements prefix to the include and link
+# paths because it is separate from the build prefix where nvcc is installed
```

Comment:
```suggestion
# For conda-build we add the host requirements prefix to the include and link
# paths because it is separate from the build prefix where nvcc is installed
# These environment variables are used in the nvcc.profile patch
```